cves/kernel/CVE-2021-38160.yml

```diff
@@ -55,7 +55,12 @@ description_instructions: |
 
   Your target audience is people just like you before you took any course in
   security
-description:
+description: When creating a buffer to make a copy, a value was read in to use
```

Comment:
The wording here is a little weird. I would start by talking about the action, then about the intricacies of the action and how it relates to the vulnerability. 
"When a copy is made on a Linux system, a buffer is used to transfer the data from one file to another. However, when the buffer is created..."